spec/configuration_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe BraspagRest::Configuration do
```

Comment:
Rebased the branch with a simple spec that asserts that the yielded object is the same as the global config :+1:.